DeepCat.hintbox.css

```diff
@@ -1,19 +1,8 @@
 /**
  * DeepCat Gadget for MediaWiki
  * @licence GNU GPL v2+
- * @author Christoph Fischer < christoph.fischer@wikimedia.de >
+ * @author Christoph Jauera < christoph.jauera@wikimedia.de >
```

Comment:
This name change should be a separate patch, but this is not critical. :+1: overall, looks good.